src/posix/platform/resolver.hpp

```diff
@@ -123,10 +123,10 @@ class Resolver : public Logger<Resolver>
     void TryRefreshDnsServerList(void);
     void LoadDnsServerListFromConf(void);
 
-    bool      mIsResolvConfEnabled    = true;
-    int       mUpstreamDnsServerCount = 0;
-    in_addr_t mUpstreamDnsServerList[kMaxUpstreamServerCount];
-    uint64_t  mUpstreamDnsServerListFreshness = 0;
+    bool         mIsResolvConfEnabled    = true;
+    int          mUpstreamDnsServerCount = 0;
```

Comment:
Same with `mUpstreamDnsServerCount`? should it be `uint16_t` or `uint_32t_t` instead of `int`?